LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 athityakumar
+Copyright (c) 2017 [Athitya Kumar](https://github.com/athityakumar/)
```

Comment:
You'll be trasferring the repo to SciRuby post GSOC so make it Athithya and SciRuby (with links).